packages/fx-core/src/core/environment.ts

```diff
@@ -216,7 +221,7 @@ class EnvironmentManager {
     if (!isMultiEnvEnabled()) {
       return ok({
         azure: {},
-        manifest: { values: {} },
+        manifest: { values: { appName: { short: "myApp" } } },
```

Comment:
Can we also consistently to use the empty string `""` as default to create a new `EnvConfig`? In `tools.ts` we set the `appName:short` to `""`, while we set it to `"myApp"` here.

Reply:
Sure, thanks!